src/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import ReactDOM from 'react-dom/client';
+import {
+  createBrowserRouter,
+  RouterProvider,
+} from 'react-router-dom';
+import store from './redux/store';
+import Root from './routes/Root';
+import ErrorPage from './routes/ErrorPage';
+import Details from './routes/Details';
+import Home from './routes/Home';
+import About from './routes/About';
+import Models from './routes/Models';
+import './index.css';
+
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <Root />,
+    errorElement: <ErrorPage />,
+    children: [
+      {
+        path: '/',
+        element: <Home />,
+      },
+      {
+        path: 'details/:id',
+        element: <Details />,
+      },
+      {
+        path: 'about',
+        element: <About />,
+      },
+      {
+        path: 'models/:id',
+        element: <Models />,
+      },
+    ],
+  },
+]);
+
```

Comment:
- [OPTIONAL] I will encourage you to extract this to an App file which is the convention.

Reply:
Thank you for letting me know about conventions @Olamarx 
I was doing just as you advised in all previous projects and this time I just saw another tutorial and thought it would be useful if I tried out a different approach.
I prefer the previous approach honestly so I totally agree with conventions.